sandbox/sandbox/Cargo.toml

```diff
@@ -0,0 +1,43 @@
+[package]
+name = "gear-sandbox"
```

Comment:
Consider
```suggestion
name = "gear-sandbox-executor"
```

Reply:
I would leave it as is since this is not an executor actually: it is a glue between wasm runtime and real host executor. It is also named `sandbox` in the old `sp-io`